lib/BillCalculator.rb

```diff
@@ -0,0 +1,54 @@
+class BillCalculator
+  attr_accessor :total, :order
+  attr_writer :meal_options
```

Comment:
A part of encapsulation is restricting direct access to an object's components. In order to increase encapsulation, we should limit the attribute readers and writers to only those we definitely need. 

Do we need this `attr_writer`? Are there other accessors we can get rid of without affecting the running of the program?